packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -603,6 +604,13 @@ class TextureLayer extends Layer {
   /// un-freezes it when it is certain that a frame with the new size is ready.
   final bool freeze;
 
+  /// Used to set the filterQuality of the texture.
```

Comment:
`Used to set the filterQuality of the texture.` doesn't provide much more information than its name `TextureLayer.filterQuality`. Maybe `The quality of sampling the texture and rendering it on screen.` is more informative. The same probably also applies to the RenderObject and the Widget.